psi4/src/psi4/libscf_solver/sad.cc

```diff
@@ -488,21 +488,33 @@ void SADGuess::get_uhf_atomic_density(std::shared_ptr<BasisSet> bas, std::shared
     // Setup JK
     std::unique_ptr<JK> jk;
 
+    std::string jk_type(options_.get_str("SAD_SCF_TYPE"));
+
+    // Handle default cases for compatibility
+    if ((jk_type == "PK") || (jk_type == "OUT_OF_CORE")) {
+        jk_type = "DIRECT";
+    }
+
     // Need a very special auxiliary basis here
-    if (options_.get_str("SAD_SCF_TYPE") == "DF") {
+    if (jk_type == "DF") {
         MemDFJK* dfjk = new MemDFJK(bas, fit);
         if (options_["DF_INTS_NUM_THREADS"].has_changed())
             dfjk->set_df_ints_num_threads(options_.get_int("DF_INTS_NUM_THREADS"));
         dfjk->dfh()->set_print_lvl(0);
         jk = std::unique_ptr<JK>(dfjk);
-    } else if (options_.get_str("SAD_SCF_TYPE") == "DIRECT") {
+    } else if (jk_type == "DIRECT") {
         DirectJK* directjk(new DirectJK(bas));
         if (options_["DF_INTS_NUM_THREADS"].has_changed())
             directjk->set_df_ints_num_threads(options_.get_int("DF_INTS_NUM_THREADS"));
         jk = std::unique_ptr<JK>(directjk);
+    } else if (jk_type == "CD") {
```

Comment:
Not sure what to do about CD however as I believe this will use disk and could cause issues.

Reply:
Oh it's not in-core?

Well, change it to direct as well I guess :P

Reply:
Unfortunately in that case the guess isn't exact, unless the Cholesky tolerance is very small.